src/MOIWrapper.jl

```diff
@@ -171,51 +234,31 @@ function MOI.copy!(cbc_optimizer::CbcOptimizer,
 
     zero_one_indices = Int[]
     integer_indices = Int[]
-    list_of_constraints = MOI.get(user_optimizer, MOI.ListOfConstraints())
-    num_rows = 0
-    for (F,S) in list_of_constraints
-        if !(MOI.supportsconstraint(cbc_optimizer, F, S))
-            return MOI.CopyResult(MOI.CopyUnsupportedConstraint,
-            "Cbc MOI Interface does not support constraints of type " * (F,S) * ".", nothing)
-        end
-
-        ci = MOI.get(user_optimizer, MOI.ListOfConstraintIndices{F,S}())
-
-        if F == MOI.SingleVariable
-            if S == MOI.ZeroOne
-                update_zeroone_indices(user_optimizer, mapping, ci, zero_one_indices)
-            elseif S == MOI.Integer
-                update_integer_indices(user_optimizer, mapping, ci, integer_indices)
-            end
-        else
-            ## Update conmap for (F,S) for F != MOI.SingleVariable
-            ## Single variables are treated by bounds in Cbc, so no
-            ## need to add a row
-            for i in 1:length(ci)
-                mapping.conmap[ci[i]] = MOI.ConstraintIndex{F,S}(num_rows + i)
-            end
-            num_rows += MOI.get(user_optimizer, MOI.NumberOfConstraints{F,S}())
-        end
-    end
+    create_indices_for_rows(cbc_optimizer, user_optimizer, mapping) # Updates mapping
+    num_rows = length(mapping.conmap)
+    # Do not create rows for bounds, but has to update mapping
```

Comment:
Nit (full sentences): "Rows are not created for bounds, but the mapping is still updated."